features/gezag-minderjarige/geen-gezag.feature

```diff
@@ -0,0 +1,21 @@
+#language: nl
+
+Functionaliteit: Over meerderjarige of overleden personen wordt geen gezag geleverd
+
+
+  Regel: Als de persoon meerderjarig is, dan is er geen sprake van gezag
+
+    Scenario: De persoon is meerderjarig
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is meerderjarig
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan heeft deze persoon geen gezaghouder
+
```

Comment:
Wat is hier de verwachte response van de API? In de oude situatie vermelden we ook het verwachte burgerservicenummer in de response. Dat deel ontbreekt hier en in plaats daarvan wordt alleen het gegeven 'geen gezaghouder' vermeld en niets over 
mogelijke andere velden die in de response voorkomen.

```feature
 Dan heeft de response een persoon met de volgende gegevens
    | naam                | waarde    |
    | burgerservicenummer | 000000036 |
```

Reply:
Zelfde als Dan heeft deze persoon geen gezag.

we verwachten dan "gezag": []